### PR TITLE
feat: add cards with links (see img)

### PR DESCRIPTION
**Pull request by lucasod3v**
image

Yup, so I added this. Content can of course be altered later as needed.

### CODE REVIEW

**Comment by kerichdev** [comment]
looks good to me, but you might wanna take a look at some of the files that you're shipping - some of them probably shouldnt be in the commit since they reference local configs and files on your pc.

secondly, have you tested this on mobile? i'm not sure how the cards will look there

lastly, we should prooooobably rework the logos in the cards to some unique style - because rn it IS a helluvaos rip-off :sob:

**Review by kerichdev — APPROVED** [review]
these look good to me

File: src/assets/PR.svg
Comment (kerichdev) [inline src/assets/PR.svg:1]: the logo needs to be redesigned to be less of a helluvaos ripoff

File: src/assets/patreonLogo.png
Comment (kerichdev) [inline src/assets/patreonLogo.png:1]: do we need the white background with it?

File: src/components/blogCard.vue
Comment (kerichdev) [inline src/components/blogCard.vue:1]: comment for all the cards - we can make them ONE component, and give them arguments for the images, text, and headers - if they share similar layout - that'll save us time down the line

File: src/components/pageHome.vue
Comment (kerichdev) [inline src/components/pageHome.vue:1]: does the layout work on mobile?

File: vite.config.js.timestamp-1723377770687-598bad10001d5.mjs
Comment (kerichdev) [inline vite.config.js.timestamp-1723377770687-598bad10001d5.mjs:1]: this probably shouldnt be here...

**Review by kerichdev — CHANGES_REQUESTED** [review]
look at the comments for the other stuff

**Comment by lucasod3v** [comment]
> looks good to me, but you might wanna take a look at some of the files that you're shipping - some of them probably shouldnt be in the commit since they reference local configs and files on your pc.
> 
> 
> 
> secondly, have you tested this on mobile? i'm not sure how the cards will look there
> 
> 
> 
> lastly, we should prooooobably rework the logos in the cards to some unique style - because rn it IS a helluvaos rip-off :sob: 

I completely forgot about mobile, my fault lol. I'll try to get it to work tmr. As for the the config files, I'll fix that too

**Comment by kerichdev** [comment]
> > looks good to me, but you might wanna take a look at some of the files that you're shipping - some of them probably shouldnt be in the commit since they reference local configs and files on your pc.
> > secondly, have you tested this on mobile? i'm not sure how the cards will look there
> > lastly, we should prooooobably rework the logos in the cards to some unique style - because rn it IS a helluvaos rip-off 😭
> 
> I completely forgot about mobile, my fault lol. I'll try to get it to work tmr. As for the the config files, I'll fix that too

no problem, take the tiem you need, req a review again after you push all the changes you need

**Review by kerichdev — APPROVED** [review]

**Comment by kerichdev** [comment]
merging as its been over 3 weeks now - propose further fixes alongside the primary development line :3

**Pull request merged**